test/taylorize.jl

```diff
@@ -129,7 +143,7 @@ using Elliptic
 
     # Pendulum integrtf = 100.0
     @testset "Integration of the pendulum and DiffEqs interface" begin
-        @taylorize function pendulum!(dx, x, p, t)
+        @taylorize function pendulum!(dx::Array{T,1}, x::Array{T,1}, p, t) where {T}
```

Comment:
Are we just testing type annotations here in the function definition, is there another reason to add them?

Reply:
Yes, you are right. Internally, we simply ignore the type annotations, but there was no test.